src/AspNet.Security.OAuth.QQ/QQAuthenticationHandler.cs

```diff
@@ -141,11 +151,38 @@ private async Task<string> GetUserIdentifierAsync(OAuthTokenResponse tokens)
                 body = body.Substring(index, body.LastIndexOf("}") - index + 1);
             }
 
-            var payload = JObject.Parse(body);
+            var payload = JsonDocument.Parse(body);
```

Comment:
This should be in a using, since it doesn't need to outlive the GetString call

Reply:
Fixed.